client/my-sites/email/email-management/style.scss

```
@@ -253,7 +253,7 @@
 	}
 
 	&.is-selected,
-	&:hover,
+	&:not(:disabled):hover,
```

It's default behaviour. And I don't see difference in UI, if we remove it. Am I missing something>
```suggestion
	&:hover,
```

---

`:hover` does actually match disabled elements as well. The `&:disabled > svg` selector takes precedence over these rules, though. Even if that's the case, I still think it makes sense to explicitly not target disabled buttons here.